microsoft-edge/devtools/performance/index.md

```diff
@@ -35,63 +35,119 @@ See also:
 
+Continue with the steps below.
 
-<!-- ------------------------------ -->
-#### Set up the demo
 
-In the following section, continuing from above, you customize the demo to make sure that your experience is relatively consistent with the screenshots and descriptions.
+<!-- ====================================================================== -->
+## Use the demo's UI to compare Slow vs. Optimized code with few or many animated icons
 
-1. Click the **Add elements** button until the blue icons move noticeably slower than before.  On a high-end machine, you can click it about 20 times.
+In the Sluggish Animation webpage, first we'll show _few_ icons, and compare selecting the demo's **Slow** option button vs. the **Optimized** option button, producing _little_ difference in the speed and smoothness of animation.  Then we'll show _many_ icons, and compare selecting the demo's **Slow** option button vs. the **Optimized** option button, producing a _great_ difference in the speed and smoothness of animation.
 
-1. Click **Optimized**.  The blue icons should move faster and more smoothly.
+Use the demo's UI, as follows:
 
-1. To better display a difference between the optimized and unoptimized versions, click the **Remove elements** button a few times and try again.  If you add too many blue icons, you could max out the CPU, and then you might not observe a major difference in the results for the two versions.
+1. Switch to the Sluggish Animation webpage.
 
-1. Click **Slow**.  The blue icons move slower and with more sluggishness again.
+   **Show few icons, and compare Slow vs. Optimized**
 
+   There are 10 blue icons by default.  The **Slow** option button is selected by default.
 
-<!-- ------------------------------ -->
-#### Record runtime performance
+1. Watch a specific blue icon move up and down.
 
-When you ran the optimized version of the page, the blue icons move faster.  Why is that?  Both versions are supposed to move the icons the same amount of space in the same amount of time.  Take a recording in the **Performance** tool to learn how to detect the performance bottleneck in the unoptimized version.
+1. Select the **Optimized** option button.
 
-Continuing from above:
+   The 10 blue icons move a little more quickly and smoothly.
+
+1. Select the **Slow** option button.
+
+   The 10 blue icons move a little more slowly.
+
+   **Show many icons, and compare Slow vs. Optimized**
+
+   Initially, the **Slow** option button is selected.
+
+1. Click the **Add elements** button around 10 times.
+
+   More blue icons are added and animated.  The more blue icons are shown, the slower and more erratic their motion.
+
+   If you add too many blue icons, such as by clicking the **Add elements** button around 20 times, you could max out the CPU, and then you might not observe a major difference in the results for the **Slow** vs. **Optimized** code.
```

Comment:
Again, avoid mentioning a number here, since that's dependent on the person's device.

On my device, clicking the button 20 times makes the Slow demo extremely slow, but the Optomized demo continues to work fine.

Reply:
If you add too many blue icons, you could max out the CPU, and then 
you might not observe a major difference in the results for the **Slow** 
vs. **Optimized** code.